cclibs/scripts/scripts.js

```diff
@@ -10,25 +10,29 @@
  * governing permissions and limitations under the License.
  */
 
-import { setLibs } from './utils.js';
-
 // Add project-wide style path here.
-const STYLES = '';
+const STYLES = '/cclibs/styles/styles.css';
 
 // Use '/libs' if your live site maps '/libs' to milo's origin.
-const LIBS = 'https://milo.adobe.com/libs';
```

Comment:
does your live site really map libs to milo's origin? 
what is your live site?

Reply:
Changed the LIBS value to pick one from milo origin to https://milo.adobe.com/libs